src/xiaomi_ble/parser.py

```diff
@@ -1289,8 +1290,11 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             elif uuid == SERVICE_HHCCJCY10:
                 if self._parse_hhcc(service_info, data):
                     self.last_service_info = service_info
+            elif uuid == SERVICE_SCALE1:
+                if self._parse_scale_v1(service_info, data):
+                    self.last_service_info = service_info
             elif uuid == SERVICE_SCALE2:
-                if self._parse_scale(service_info, data):
```

Comment:
If we add more we can make a dispatcher based on the uuid to the function

Reply:
Example

https://github.com/esphome/aioesphomeapi/blob/2df4effef31fcff786f86c020c4385f653b480ec/aioesphomeapi/_frame_helper/noise.py#L355

Reply:
There have been only two formats for MiScale for several years now, which is used for all scales so far (afaik 5types). I don’t expect they will make a new format anytime soon, so I propose we keep it as it is for now.